x-pack/plugins/maps/public/reducers/default_map_settings.ts

```diff
@@ -4,11 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { MAX_ZOOM, MIN_ZOOM } from '../../common/constants';
+import { INITIAL_LOCATION, MAX_ZOOM, MIN_ZOOM } from '../../common/constants';
 import { MapSettings } from './map';
 
 export function getDefaultMapSettings(): MapSettings {
   return {
+    initialLocation: INITIAL_LOCATION.LAST_SAVED_LOCATION,
+    initialLat: 0,
+    initialLon: 0,
+    initialZoom: 2,
```

Comment:
this is a similar issue as we have with the style-properties. but this multiplexes these values into a single flat object, although not all of them are relevant.

consider three types:

```
type LastSaved {
   type: string
}

type Fixed {
   type: string,
   initialLon: number,
   initialLat: number,
   initialZoom: number,
}

type Browser {
   type: string,
   initialZoom: number
}
```

The biggest advantage is that this avoid unused cruft to accumulate in the saved-object. (<---> easy UX sugar). (can flip either way, but I think the above communicates better how everything relates).